libecho/docs/receiver.md

```diff
@@ -0,0 +1,50 @@
+# Receiver
```

Comment:
Spoko, ale jakbyś chciał to ja bym się chętnie dowiedziała jeszcze dokładniej jak to działa: jak np dostosowywujemy lim itd.